api/resource.rb

```diff
@@ -161,10 +161,16 @@ class IamPolicy < Api::Object
       # boolean of if this binding should be generated
       attr_reader :exclude
 
+      # Character that separates resource identifier from method call in URL
+      # For example, PubSub subscription uses {resource}:getIamPolicy
+      # While Compute subnetwork uses {resource}/getIamPolicy
+      attr_reader :method_name_separator
```

Comment:
Is this only used for IAM? If so, we might want to put "IAM" in the name somewhere.

Reply:
Lols, never mind. Inside the IAM block.